testing/sdk_networks.py

```diff
@@ -7,6 +7,8 @@
 import logging
 import shakedown
 import sdk_cmd
```

Comment:
I know it's not documented anywhere, but let's first import standard library packages, and then the `testing` modules (`sdk_*`) and then the local test modules (if any).

Reply:
After stdlib imports, but before local imports.